Bio/Blast/Applications.py

```diff
@@ -984,26 +984,26 @@ def __init__(self, cmd="rpsblast", **kwargs):
                     """,
                     equate=False),
             _Option(["-best_hit_overhang", "best_hit_overhang"],
-                    """Best Hit algorithm overhang value (recommended value: 0.1)
+                    r"""Best Hit algorithm overhang value (recommended value: 0.1)
 
                     Float between 0.0 and 0.5 inclusive.
 
                     Incompatible with: culling_limit.""",
                     equate=False),
             _Option(["-best_hit_score_edge", "best_hit_score_edge"],
-                    """Best Hit algorithm score edge value (recommended value: 0.1)
+                    r"""Best Hit algorithm score edge value (recommended value: 0.1)
 
                     Float between 0.0 and 0.5 inclusive.
 
                     Incompatible with: culling_limit.""",
                     equate=False),
             # General search options:
             _Option(["-comp_based_stats", "comp_based_stats"],
-                    """Use composition-based statistics.
+                    r"""Use composition-based statistics.
 
                     D or d: default (equivalent to 0 )
                     0 or F or f: Simplified Composition-based statistics as in
-                                 Bioinformatics 15:1000-1011, 1999
+                    \             Bioinformatics 15:1000-1011, 1999
```

Comment:
Again why the slash. I think the simplest way to make this look nice as RST is to make each of these into its own paragraph by adding blank lines?

Reply:
I suspect my IDE was inserting extraneous white space and triggering W293. I've amended those particular files and reformatted them for nicer output.